### PR TITLE
[AWS] Add description to the Data Granularity setting

### PR DESCRIPTION
**Pull request by zmoog**
This is just a draft to kick start a conversation.

## What does this PR do?

Expand the documentation related to Data Granularity, an advanced configuration setting on the CloudWatch metrics based integrations.

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [ ] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Related issues

-

## Screenshots

CleanShot 2023-05-16 at 06 54 08@2x

### CODE REVIEW

File: packages/aws/data_stream/billing/manifest.yml [hunk 1]
```diff
@@ -13,6 +13,7 @@ streams:
       - name: data_granularity
         type: text
         title: Data Granularity
+        description: "Granularity of the data points returned in the Collection Period. Example values are `1m` or `5m`, see Golang [time.ParseDuration](https://pkg.go.dev/time#ParseDuration) for more details. Allows you to query CloudWatch asking “Give me the four 5-minute (Data Granularity) data points for the last 20 minutes (Collection Period)“. To learn more about how to use data granularity, see [Metrics collection and cost considerations](https://docs.elastic.co/integrations/aws#metrics-collection-and-cost-considerations)."
```
Comment (girodav) [inline packages/aws/data_stream/billing/manifest.yml:16]: Not sure if `Allows you to query CloudWatch asking “Give me the four 5-minute (Data Granularity) data points for the last 20 minutes (Collection Period)“` won't be confusing for users. 
Maybe we should just explain that the value is passed down as "Period" for MetricDataQueries and direct the users to the AWS docs https://docs.aws.amazon.com/AmazonCloudWatch/latest/APIReference/API_MetricDataQuery.html

Comment (girodav) [reply packages/aws/data_stream/billing/manifest.yml:16]: I wonder if we should also mention that it cannot be lower than the frequency Clouwatch (CW) publishes events. We cannot make a metric more "granular" than what is available on CW.

Comment (kaiyan-sheng) [inline packages/aws/data_stream/billing/manifest.yml:16]: I agree and this is a hard one to explain. What do you think about this @zmoog @girodav : 
```suggestion
        description: Granularity of the returned data points. Example values are `1m` , `5m` and etc, see Golang [time.ParseDuration](https://pkg.go.dev/time#ParseDuration) for more details. This parameter works with `period` together to define how frequently the metrics are collected and what the time difference is between returned data points. `data_granularity` value cannot be lower than the frequency CloudWatch publishes events and also cannot be higher than `period`. To learn more about how to use data granularity, see [Metrics collection and cost considerations](https://docs.elastic.co/integrations/aws#metrics-collection-and-cost-considerations).
```

[the remark this answers is not in the document]
Comment (kaiyan-sheng) [reply packages/aws/data_stream/billing/manifest.yml:16]: We probably should add the documentation for granularity under the main integration https://docs.elastic.co/integrations/aws and then just keep the description in each data stream short and with a reference to the main doc. I think this is getting too long to display when adding each integration on the UI.

**Pull request closed without merging**